src/views/TechnicalAssistance/RequestForm/Basic.tsx

```diff
@@ -431,6 +435,34 @@ function Basic({
                     );
                   })}
                 </Dropdown>
+                {field.value === 'OTHER' && (
+                  <Controller
+                    name="whereInProcessOther"
+                    control={control}
+                    // eslint-disable-next-line no-shadow
+                    render={({ field, fieldState: { error } }) => (
```

Comment:
A nitpick but it might be better to alias the shadowed variables.  Would provide clarity and potential developer error

```
render={({ field: fieldOther, fieldState: { error: errorOther } }) => (
```

Reply:
If this is really just considered a nitpick, then I would leave it.

This has come up before. Basically decided to alias when it's actually necessary. https://github.com/CMSgov/easi-app/pull/1820#discussion_r1015655527